lib/AnomalyTree.js

```diff
@@ -16,7 +16,11 @@
 
 "use strict";
 
-var NODEJS =  process && process.RUNNING_IN_NODEJS === 'true';
+if (typeof process !== 'undefined') {
+  var NODEJS =  process && process.RUNNING_IN_NODEJS === 'true'; 
+} else {
+  var NODEJS = false;
+}
```

Comment:
why not just:

```nodejs
var NODEJS = process !== 'undefined' && process && process.RUNNING_IN_NODEJS;
```

sans `if`...

Reply:
because in javascript `process` does not exist and this throws an error

Reply:
```
>> (typeof foo !== 'undefined') && foo && foo.BAR === 'true'
>> false
```
might be the trick in case you want to change the code.